compute/instances.go

```diff
@@ -103,6 +104,39 @@ func (gmi *GetInstanceInput) Validate() error {
 	return nil
 }
 
+//
+type CountInstancesInput struct{}
+
+//
+func (c *InstancesClient) Count(ctx context.Context, _ *CountInstancesInput) (int, error) {
+	fullPath := path.Join("/", c.client.AccountName, "machines")
+
+	reqInputs := client.RequestInput{
+		Method: http.MethodHead,
+		Path:   fullPath,
+	}
+
+	response, err := c.client.ExecuteRequestRaw(ctx, reqInputs)
```

Comment:
I think you need to check `err != nil` here

Reply:
Well spotted! Fixed.